src/Entity/TranslationRequestInterface.php

```diff
@@ -147,4 +147,12 @@ public function getData(): array;
    */
   public function setData(array $data): TranslationRequestInterface;
 
+  /**
+   * Retrieves the Translation request log entity.
+   *
+   * @return \Drupal\oe_translation\Entity\TranslationRequestLogInterface
+   *   The Translation request log entity.
+   */
+  public function getLog(): TranslationRequestLogInterface;
```

Comment:
Let's also have a method called `log()` which takes a TranslationLog entity as parameter and sets it next in the logs field.